### DISCUSSION TITLE
My Personal Feedback- and Wishlist

### DISCUSSION THREAD

**Discussion by psteinroe** [supabase/supabase#9235] [Show and tell]
Hey,

this discussion only serves as my personal feedback board and wishlist where I track issues, thoughts and ideas that come up when working with supabase on a larger scale project.

**Content for larger scale projects**
Right now, most content is around hobby projects. It would be great to expand towards more sophisticated topics such as pg_tap, RLS, multi-tenancy, etc...

**Seamless sql development for teams**
I am really looking forward to the recently announced Postgres branching, but until then the team should invest into a smoother development experience with what is there
- https://github.com/supabase/cli/issues/192
- Best case would be that every developer would have just a single migration file locally that applies the latest state from a remote project. Like a "clone" of the db.

**pg_net**
- https://github.com/supabase/pg_net/issues/73

**Current State of Realtime**
While its great to see how fast the team ships new features, everything around realtime feels a bit rushed. The DX on the product is by far the worst of supabase, and the new multiplayer features have been shipped without its basic postgres_change functionality being carried along properly. I am aware of an ongoing effort around realtime, but there are few things that are very frustrating, e.g. missing types or not updating the docker image in the cli. 
- https://github.com/supabase/realtime-js/issues/159
- https://github.com/supabase/realtime/issues/222
- https://github.com/supabase/supabase-js/issues/559

**Queues / Webhooks**
- https://github.com/supabase/realtime/issues/299

**Transparency + Self-Service with components and extensions**
- https://github.com/supabase/supabase/discussions/9226
- https://github.com/supabase/supabase/discussions/9227

**Alerts**
- https://github.com/supabase/supabase/discussions/9228
- https://github.com/supabase/supabase/discussions/9232
- https://github.com/supabase/supabase/discussions/9231

**Postgres as application server**
- https://github.com/supabase/supabase/discussions/9233
- https://github.com/supabase/supabase/discussions/9219
- https://github.com/supabase/supabase/discussions/9220

**Other**
- https://github.com/supabase/supabase/discussions/9218
- https://github.com/supabase/supabase/discussions/9234

**Comment by thorwebdev** [comment]
thanks @psteinroe for taking the time to create all these and collate this list! 💚 I'm going through and flagging these with the relevant teams 👍
